### DISCUSSION TITLE
the farm stopped seeing all the plots

### DISCUSSION THREAD

**Discussion by lmnch1** [Chia-Network/chia-blockchain#11235] [Support]
Hello!
Today, at the release of the chia 1.3.3 blockchain, the farm stopped seeing all plots, decided to update the release to the chia 1.3.4 blockchain, but after the update, the problem did not disappear.
 Also, when trying to delete a directory containing plots, an error occurs "Error
[Error 13] Permission denied. You are trying to access a file/directory without having the necessary permissions. Most likely, the problem is in one of the pst folders in your config.yaml."
Up to this point, there was no such problem either.

**Comment by loppefaaret** [comment]
What OS are you on ? and have you ensured that you do have permissions to the folder that you are trying to access ?
